extensions/wikia/Recirculation/js/incontent.js

```diff
@@ -0,0 +1,187 @@
+/*global require*/
+require([
+	'jquery',
+	'wikia.window',
+	'wikia.abTest',
+	'wikia.nirvana',
+	'wikia.mustache',
+	'ext.wikia.recirculation.tracker',
+	'ext.wikia.recirculation.utils'
+], function ($, w, abTest, nirvana, Mustache, tracker, utils) {
+	// Currently only showing for English communities
+	if (w.wgContentLanguage !== 'en') { return; }
```

Comment:
What do you think about adding one more dependency in form of `wikia.log` and using logger with all those early returns. This way when somebody comes to you and asks why she doesn't see the widget on a page you'll be able to easily add `?log_level=debug&log_group=...` to the page URL and see in console what is the reason. If it's expected because it gets to one of those early returns or if it's a unhandled "exception". You can find a lot of examples how to use `wikia.log` module in `AdEngine` extension for example in [AdInContentPlayer.js](https://github.com/Wikia/app/blob/3503f5f49310c5cf7912c95fd61d42b42d5ea7c2/extensions/wikia/AdEngine/js/AdInContentPlayer.js#L48).